services/121-service/src/program-financial-service-provider-configurations/program-financial-service-provider-configurations.repository.ts

```diff
@@ -67,6 +67,26 @@ export class ProgramFinancialServiceProviderConfigurationRepository extends Repo
     return response;
   }
 
+  // This methods specfically does not throw as it also used to check if the property exists
```

Comment:
I ended up adding this function. Because I specifically needed a function to test if a property exists for the Excel service instead of a function that throws if something does not exist. If we refactor excel maybe we can reconsider having this here